src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,89 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
+* [Dive deeper in stack traces](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#stack-traces)
+* [Identify gaps in your alerting ecosystem](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#coverage-gaps)
+* [Create synthetic monitors](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#)
+* [Understand your dashboards](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#dashboards)
+* [Check the status of support tickets](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai/#support)
+
+<Video
+ type="youtube"
+ id="7vBhF4Ph_LA?si=UyaTG9LwSfkqVUU5"
+/>
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
 
-We help you get to the heart of your system by offering quick and easy ways to ask questions: AI chat, and the NRQL console. 
+We help you get to the heart of your system by offering quick and easy ways to ask questions. You can chat with our AI and interact with it in the NRQL console or work with some of our AI tools directly in the UI.
 
-### Ask AI questions [#ask-AI]
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the <DoNotTranslate>**Ask AI**</DoNotTranslate> button available on the top right of most New Relic platform pages.
 
-New Relic AI provides a variety of capabilities to help you:
-* Transform natural language into queries (NRQL/Nerdgraph)
-* Detect anomalies
-* Analyze error logs
-* Understand stack traces
-* Leverage our docs to understand New Relic platform concepts
+### Chat with AI [#ask-AI]
 
-Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store, and you start receiving alerts that response times for your `Checkout-service` are lagging. If you don't have time to dig into the issue, you can just ask New Relic AI "What's on fire?" or "What should I handle first today?" New Relic AI can tell you what's broken in your system and help you explore data with charts and tables.
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the <DoNotTranslate>**Ask AI**</DoNotTranslate> button available on the top right of most New Relic platform pages.
 
-## Sample questions [#sample-questions] 
+Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store and start receiving alerts that response times for your `Checkout service` are lagging. If you don't have time to investigate the issue, you can simply ask New Relic AI, "What's on fire?" or "What should I handle first today?" New Relic AI can tell you what's broken in your system and help you explore the data with charts and tables.
```

Comment:
Again, I think 'natural language' is the correct term here over 'plain language'.

Reply:
I know that it's technically correct to have the punctuation of the sentence inside the quotations, but I've always thought it read _very_ awkwardly with question marks. Maybe make it into a 3 part list that ends with ", or something similar."

Reply:
Fixed this!